engine/src/main/java/io/graphqlcrud/SQLQueryBuilderVisitor.java

```diff
@@ -271,67 +273,130 @@ public void visitArgument(Field field, GraphQLFieldDefinition definition, GraphQ
 
     public Condition visitStringValue(Value v, org.jooq.Field left, String conditionName) {
```

Comment:
Change the parameter `Value v' to `String` and remove the class castings below. When you are calling this method you already know the type just cast once there. Same for others below too